tes-adapter/scripts/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# start from base
+FROM centos:7
+# install
+RUN apt-get -y install java-1.8.0-openjdk
+# copy script
+ADD scripFromEuv /init
```

Comment:
what is `scriptfromEuv`?

Reply:
@ilya-ugr but this script builds jar file, and doesn't run it.
So you need to copy it to your image, run it to get jar available on image, but command in CMD should be like:
```
CMD['java', '-jar', '<path to jar>']
```
FYI it's better to put jar file in directory `/opt` on an image